### PR TITLE
Resolve console warnings

### LINKED ISSUE #260: Warning: validateDOMNesting(...): <h1> cannot appear as a child of <h3>
The following warning should be addressed:

image

### LINKED ISSUE #261: Warning: Each child in a list should have a unique "key" prop
The following warning should be addressed:

image

### LINKED ISSUE #267: Warning: Can't perform a React state update on an unmounted component
image

### LINKED ISSUE #269: Warning:  A locale object was not found for the provided string ["de-ch"]
image

### PR DESCRIPTION
**Pull request by ningxie1991**
Fixes #260, #261, #267 and #269

### CODE REVIEW

File: src/components/DateRangeSelector.tsx [hunk 1]
```diff
@@ -106,7 +106,6 @@ export const DateRangeSelector = ({ dateRange }: Props) => {
               endDate={endDate}
               minDate={minimumDate}
               maxDate={endDate}
-              locale='de-ch'
```
Comment (ningxie1991) [inline src/components/DateRangeSelector.tsx]: #269

File: src/components/NamedCard.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const NamedCard = ({
   return (
     <Card namedCardStyle={style}>
       <Title>
-        <h1 className='my-0'>{title}</h1>
```
Comment (ningxie1991) [inline src/components/NamedCard.tsx:129]: #260

File: src/pages/FocusPage.tsx [hunk 3]
```diff
@@ -106,19 +107,23 @@ export const FocusPage = ({
...
   const [wasteWaterData, setWasteWaterData] = useState<WasteWaterDataset | undefined>(undefined);
   useEffect(() => {
+    let isMounted = true;
     if (!variant.name) {
       return;
     }
     getData({
       country,
-    }).then(dataset => dataset && setWasteWaterData(filter(dataset, variant.name)));
+    }).then(dataset => isMounted && dataset && setWasteWaterData(filter(dataset, variant.name)));
```
Comment (ningxie1991) [inline src/pages/FocusPage.tsx]: #267

File: src/charts/Metrics.tsx [hunk 4]
```diff
@@ -102,8 +102,8 @@ export const ChartAndMetrics = ({ children, metrics, title, metricsTitle }: Char
         <div className='w-full h-full'>{children}</div>
         <MetricsWrapper>
           {metricsTitle && <h3>{metricsTitle}</h3>}
-          {metrics.map(mProps => (
-            <Metric {...mProps} />
+          {metrics.map((mProps, index) => (
+            <Metric key={index} {...mProps} />
```
Comment (ningxie1991) [inline src/charts/Metrics.tsx]: #261

File: src/components/KnownVariantsList/KnownVariantsListSelection.tsx [hunk 5]
```diff
@@ -46,8 +46,8 @@ export const KnownVariantsListSelection = ({ variantLists, selected, onSelect }:
     <Dropdown>
       <Dropdown.Toggle as={CustomToggle}>{selected}</Dropdown.Toggle>
       <Dropdown.Menu as={CustomMenu}>
-        {variantLists.map(({ name }) => (
-          <Dropdown.Item active={name === selected} onClick={() => onSelect(name)}>
+        {variantLists.map(({ name }, index) => (
+          <Dropdown.Item key={index} active={name === selected} onClick={() => onSelect(name)}>
```
Comment (ningxie1991) [inline src/components/KnownVariantsList/KnownVariantsListSelection.tsx:50]: #261

File: src/pages/FocusPage.tsx [hunk 6]
```diff
@@ -57,8 +57,9 @@ const deepFocusPaths = {
   wasteWater: '/waste-water',
 };
 
-const createDivisionBreakdownButton = (setter: (show: boolean) => void) => (
+const createDivisionBreakdownButton = (key: string, setter: (show: boolean) => void) => (
   <Button
+    key={key}
```
Comment (ningxie1991) [inline src/pages/FocusPage.tsx:54]: #261

**Comment by chaoran-chen** [comment]
Hi @ningxie1991, thanks a lot for solving these issues! Would it be okay for you if we wait a bit and merge #264 first?

**Comment by ningxie1991** [comment]
> Hi @ningxie1991, thanks a lot for solving these issues! Would it be okay for you if we wait a bit and merge #264 first?

@chaoran-chen I resolved all the merging conflicts and this is ready to be merged now.

**Comment by chaoran-chen** [comment]
Thanks @ningxie1991, I'll review it in the afternoon/evening!

**Review by chaoran-chen — APPROVED** [review]
Looks good. Thanks!

**Review by spacephoton — APPROVED** [review]

**Pull request merged**